lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
Is there a cleaner way to do this?

Looks like the bindings module might [automatically detect platform, arch, etc.](https://github.com/TooTallNate/node-bindings/blob/master/bindings.js) depending on the directory structure.

I think if we make the directory structure  `module_root/compiled/version/platform/arch/bindings` then we can just use `require('bindings')()`.